tests/text-translator/run_tests.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# run a given script in the same directory as the current script
+run_and_check_output() {
```

Comment:
Assuming this is going to be used in other tests, you could move this helper to another generic helper file and source it to include it.